File: Solutions/1431. Kids With the Greatest Number of Candies/1431.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        maxIndex = 0
        for i in range(1, len(candies)):
            if(candies[maxIndex] < candies[i]):
                maxIndex = i
        ans = []
        for i in range(len(candies)):
            if candies[i]+extraCandies >= candies[maxIndex]:
                ans.append(True)
            else:
                ans.append(False)
        return ans
        

File: Solutions/1431. Kids With the Greatest Number of Candies/1431.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int maxIndex = 0;
        for(int i=1;i<candies.length;i++){
            if(candies[maxIndex] < candies[i]){
                maxIndex = i;
            }
        }
        List<Boolean> ans = new ArrayList<>();
        for(int i=0;i<candies.length;i++){
            if(candies[i] + extraCandies >= candies[maxIndex]){
                ans.add(true);
            }else {
                ans.add(false);
            }
        }
        return ans;
    }
}